CorsixTH/Lua/room.lua

```diff
@@ -779,13 +779,15 @@ function Room:crashRoom()
       table.remove(self.world:getLocalPlayerHospital().emergency_patients, humanoid.is_emergency)
     end
     humanoid:die()
+    humanoid:despawn()
     self.world:destroyEntity(humanoid)
   end
 
   -- Remove all humanoids in the room
   for humanoid, _ in pairs(self.humanoids) do
     remove_humanoid(humanoid)
   end
+  self.humanoids = nil
```

Comment:
nit pick, but I think I'd prefer you set this back to the empty set rather than nil, similar to what happens to `humanoids_enroute` in deactivate